src/coinbase/validator.ts

```diff
@@ -0,0 +1,64 @@
+import { GetValidator200ResponseValidator, ValidatorListData } from "../client";
+import { Coinbase } from "./coinbase";
+
+/**
+ *
+ */
+export class Validator {
+  /**
+   * Returns the list of Validators.
+   *
+   * @param networkId - The network ID.
+   * @param assetId - The asset ID.
+   * @param status - The status to filter by.
+   * @returns The list of Validators.
+   */
+  public static async list(
+    networkId: string,
+    assetId: string,
+    status?: string,
+  ): Promise<ValidatorListData> {
+    let validators: ValidatorListData = {};
+
+    const response = await Coinbase.apiClients.validator!.listValidators(
+      networkId,
+      assetId,
+      status,
+    );
+
+    if (networkId.startsWith("ethereum") && assetId === "eth") {
+      validators = response.data.data;
+    }
+
+    return validators;
+  }
+
+  /**
+   *
+   * Returns the details of a specific validator.
+   *
+   * @param networkId - The network ID.
+   * @param assetId - The asset ID.
+   * @param validatorAddress - The address of the validator for which to fetch the details.
+   * @returns The requested validator details.
+   */
+  public static async fetch(
+    networkId: string,
+    assetId: string,
+    validatorAddress: string,
+  ): Promise<GetValidator200ResponseValidator> {
```

Comment:
we may want to use an alias for `GetValidator200ResponseValidator`

Reply:
Done 👍